.github/workflows/build-and-test-workflow.yml

```diff
@@ -95,7 +95,7 @@ jobs:
 
       # Step: CMake build
       - name: Build
-        run: cmake --build ${{ steps.paths.outputs.build_dir }} --config ${{ inputs.build_type }} -j
+        run: cmake --build ${{ steps.paths.outputs.build_dir }} --config ${{ inputs.build_type }} -j 4
```

Comment:
next time i suggest you make this part of a separate pr

Reply:
This was just a quick test for initial timings, I will revert to original state.